lib/router/app_router.dart

```diff
@@ -72,6 +72,20 @@ class AppRouter extends $AppRouter {
         AutoRoute(
           page: EventSelectingRoute.page,
         ),
+        AutoRoute(
+          page: OnboardingWrapperRoute.page,
+          children:[
+            AutoRoute(
+              page: OnboardingUsernameRoute.page,
```

Comment:
should add path

Reply:
Actually we don't need to add path for every route here. When navigating we should use `context.router.push(PageRoute)` instead of using `AutoRoute.of(context).push('pagepath'). Using raw string path will causing change to have unwanted bug

Reply:
my point is here is not only for navigateNamed only,  but maybe for deeplink , but yeah, can skip it for now